src/brod_transaction.erl

```diff
@@ -0,0 +1,343 @@
+%%%
+%%%   Copyright (c) 2014-2021 Klarna Bank AB (publ)
```

Comment:
this project is no longer under Klarna org, you can put your name here if you want.
Or "kafka4beam and contributors"
```suggestion
%%%   Copyright (c) 2023 @axs-mvd and contributors
```